libs/functions/generate_api.py

```diff
@@ -143,6 +146,10 @@ def _generate_state_timeseries_row(json_data_row):
 
 
 def _generate_county_timeseries_row(json_data_row):
+    tested = _get_or_none(json_data_row[CDSDataset.Fields.TESTED])
+    cases = _get_or_none(json_data_row[CDSDataset.Fields.CASES])
+    negative = tested and cases and (tested - cases)
```

Comment:
sorry, what's the logic?

Reply:
negative is null unless we have tested and cases, in that case it's tested - cases